src/components/Post/ActionsBar/Reactionbar/TooltipContent.tsx

```diff
@@ -0,0 +1,49 @@
+// Copyright 2019-2025 @polkassembly/polkassembly authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+/* eslint-disable @next/next/no-img-element */
+import React from 'react';
+import { UserProfileImage } from 'pages/api/v1/auth/data/userImage';
+import ImageComponent from '~src/components/ImageComponent';
+import Loader from '~src/ui-components/Loader';
+
+interface TooltipContentProps {
+	users: UserProfileImage[];
+	usernames: string[];
+	isLoading: boolean;
+}
+
+const TooltipContent: React.FC<TooltipContentProps> = ({ users, usernames, isLoading }) => {
+	const allUsers = users.map((user, index) => ({
+		id: user.id,
+		image: user.image,
+		username: usernames[index]
+	}));
+
+	return (
+		<div className='max-h-24 w-min overflow-auto'>
+			{isLoading ? (
+				<Loader className='h-7 w-7' />
+			) : (
+				<div>
+					{allUsers.map((user) => (
+						<div
+							key={user.id}
+							className='mb-[6px] flex items-center gap-[6px]'
+						>
+							<ImageComponent
+								src={user.image}
```

Comment:
make it a Link if someone clicks on that it should redirect to a new tab and the profile should be open against this username